backend/src/main/java/de/bund/digitalservice/ris/norms/config/OtcObsConfig.java

```diff
@@ -66,6 +73,20 @@ public S3Client publicS3Client() throws URISyntaxException {
     return createS3Client(publicAccessKeyId, publicSecretAccessKey);
   }
 
+  /**
+   * Creates a {@link S3Client} for accessing the eVerkündungs S3 bucket.
+   *
+   * <p>This bean is only available in the "staging", "uat", and "production" profiles.</p>
+   *
+   * @return an instance of {@link S3Client} configured for the eVerkündungs bucket.
+   * @throws URISyntaxException if the endpoint URI is malformed.
+   */
+  @Bean(name = "eVerkuendungS3Client")
+  @Profile({ "staging", "uat", "production" })
```

Comment:
S3 client should only be configured for uat right?

Reply:
We don't think so:
- it is a general feature of our application that will (atleast at some point in time) also be needed on production
- and thus we might also have a need to check something with that feature on staging (eg. test a case that isn't working

I can also imagine us changing our current upload to also use this bucket once we changed it to support zip files, as the further processing should be the same.

Also in general I dont think our environments should differ too much - and that the portal-prototype thing is only enabled in one environment is the big exception and only okay as it is really only a temporary thing